content/en/docs/admin/admission-controllers.md

```diff
@@ -532,21 +517,78 @@ for more information.
 
 ### PodTolerationRestriction
 
-This admission controller first verifies any conflict between a pod's tolerations and its
-namespace's tolerations, and rejects the pod request if there is a conflict.
-It then merges the namespace's tolerations into the pod's tolerations.
-The resulting tolerations are checked against the namespace's whitelist of
-tolerations. If the check succeeds, the pod request is admitted otherwise
-rejected.
+This admission controller defaults and limits what tolerations may be used on Pods in a given namespace.
```

Comment:
This doesn't read well "This admission controller defaults and" ... 'defaults' to what?